jdaviz/configs/default/plugins/model_fitting/fitting_backend.py

```diff
@@ -0,0 +1,54 @@
+from asteval import Interpreter
+
+from specutils.spectra import Spectrum1D
+from specutils.fitting import fit_lines
+
+__all__ = ['fit_model_to_spectrum']
+
+
+def fit_model_to_spectrum(spectrum, component_list, expression):
```

Comment:
```suggestion
def fit_model_to_spectrum(spectrum, component_list, expression, run_fitter=False):
```

Reply:
Initiating the fitting functionality would be a separate UI-interection. This interaction probably should be a keyword argument that is set when the user selects the "Fit model" button in the UI. Otherwise, this function should compose the compound model without fitting and return just the initial estimations. This allows for easy iteration in the case where the user wants to just manually enter parameter values.